### ISSUE TITLE
UFO Ending not saving?

### ISSUE THREAD

**Issue by GlitchyReal** [elishacloud/Silent-Hill-2-Enhancements#984]
From what I recall and am told, the original console versions would record the four basic endings, the BfaW ending, and the two joke endings in the final result screen as 4 (+3).

On my 10 Star run, I had 4 (+2) and while that is all that's required for 10 Stars, I should have +3. I have gotten all the endings multiple times and saved results after. I also do not have the UFO Ending in the Movies menu.

**Comment by Polymega** [comment]
I don't believe there's any issues with recording the UFO ending. Did you ensure you loaded a previous Game Result file before doing your UFO run? The UFO ending won't be in the Movies menu as it's technically not a movie but a series of still images (think of a PowerPoint presentation). Only Bink videos are housed in the Movies menu.

**Comment by GlitchyReal** [comment]
I did make sure to play an extra New Game after the Game Result and loaded a Game Result save with 4 (+2). I then completed a run with the UFO ending and the endings tally did not update to 4 (+3).

I also run this game regularly with the Dog ending (and saving aftewards) as well as ocassionally the BfaW ending. I got the UFO Ending last and is the only one I noticed it being at 4 (+2). I reran the game after getting the final ending (UFO) for 10 Stars and finished with the Dog Ending for 4 (+2). I ran BfaW and saved. I played the main scenario again to still have 4 (+2).

I did not import save file either.

If it isn't the UFO ending, one of the bonus endings isn't counting on my file.
